.github/config/issue-labler/labler-v1.yml

```diff
@@ -1,21 +1,15 @@
-area.ui:
-  - '- \[ ?(?:x|X) ?\] UI'
-area.workbench:
-  - '- \[ ?(?:x|X) ?\] Workbench'
-area.publishing:
-  - '- \[ ?(?:x|X) ?\] Publishing'
-area.pods:
-  - '- \[ ?(?:x|X) ?\] Pods'
-area.markdown:
-  - '- \[ ?(?:x|X) ?\] Markdown'
-area.seeds:
-  - '- \[ ?(?:x|X) ?\] Seed'
+area.onboard:

```

Comment:
You should not edit this file. 
Copy the content into `labler-v2.yml`

Reply:
ah, good to know, thanks.